plugins/extractors/optimus/optimus_test.go

```diff
@@ -118,7 +118,7 @@ func (c *mockClient) GetJobTask(
 }
 
 func setupExtractExpectation(ctx context.Context, client *mockClient) {
-	client.On("Connect", ctx, validConfig["host"]).Return(nil).Once()
+	client.On("Connect", ctx, validConfig["host"], 0).Return(nil).Once()
```

Comment:
have we tested it for any value other tha 0 mb??

Reply:
we can put other value, but right now there is no way of test the impact unless we have lots of testdata. @GrayFlash

Reply:
okay, seems good for now then